bitnami/mongodb/templates/replicaset/statefulset.yaml

```diff
@@ -432,11 +432,21 @@ spec:
           {{- end }}
           env:
           {{- if .Values.auth.enabled }}
+          {{- if not (or .Values.metrics.username .Values.metrics.password ) }}
```

Comment:
Same here, if someone leaves metrics.password with a value but has metrics.username empty, that could cause some issue. We could just check for metrics.username instead.

Reply:
I removed `.Values.metrics.password` from the query